app-ui/src/bundles/Auth/containers/ActivateAccountContainer.js

```diff
@@ -4,24 +4,33 @@ import { history } from 'modules/LocationModule';
 import { sendActivationEmail } from 'bundles/Auth/modules/ActivateAccountModule';
 import ActivateAccount from 'bundles/Auth/components/ActivateAccount';
 import config from 'config/index';
+import { getActivateAccountEmail } from 'bundles/Auth/selectors/AuthSelectors';
 
 /**
  * Maps the state properties to the React component `props`.
  *
  * @param {Object} state The application state.
  * @returns {Object} The props passed to the react component.
  */
-const mapStateToProps = state => ({
-  ...state.auth.activateAccount,
-});
+const mapStateToProps = state => getActivateAccountEmail(state);
+
+/**
+ * Attaches callback to on activation request success
+ * @param r
+ */
+export const onActivationSent = (r) => {
```

Comment:
Maybe you could write `request` instead of `r` and add a proper documentation. I think this must be done for the other containers as well.

Reply:
Corrrected